### ISSUE TITLE
Too many "The number of training batches is smaller than the logging interval Trainer(log_every_n_steps=50)" Warning

### ISSUE THREAD

**Issue by Kevin-Chen0** [ourownstory/neural_prophet#1036]
There are over 600 warnings in the unit tests, where the mass majority of them are this message:

```
The number of training batches (10) is smaller than the logging interval Trainer(log_every_n_steps=50). Set a lower value for log_every_n_steps if you want to see logs for the training epoch.
```

![image](https://user-images.githubusercontent.com/13785749/205451513-c6a6a785-dae8-4204-9c89-592649154b20.png)

Either resolve the logging interval Trainer (e.g. make it more dynamic), and disable this warning all together.

**Comment by karl-richter** [comment]
The PossibleUserWarning should be silenced in the pytests. Can it be that you just run `pytest` in the command line? You would need to use `pytest tests` so that pytest picks up the `pytest.ini` file under `tests/`. Let me know if that reduces the warnings.

**Comment by karl-richter** [comment]
@Kevin-Chen0 the warning you mentioned in the title is intentional since it informs the user the learning rate finder might not work properly since the provided dataset contains too few data.

**Comment by karl-richter** [comment]
We also explicitly encourage the usage of `pytest tests` in the [contributing guidelines](https://github.com/ourownstory/neural_prophet/blob/main/CONTRIBUTING.md#testing-and-code-coverage). I would close it with that, let me know if you think otherwise.
